docs/design/acceptance_test.md

```diff
@@ -0,0 +1,132 @@
+# Acceptance test for kube-arangodb operator on specific Kubernetes platform
+
+This acceptance test plan describes all test scenario's that must be executed
+succesfully in order to consider the kube-arangodb operator production ready
+on a specific Kubernetes setup (from now on we'll call a Kubernetes setup a platform).
+
+## Platform parameters
+
+Before the test, record the following parameters for the platform the test is executed on.
+
+- Name of the platform
+- Version of the platform
+- Upstream Kubernetes version used by the platform
+- Number of nodes used by the Kubernetes cluster
+- `StorageClasses` provided by the platform (run `kubectl get storageclass`)
+- Does the platform use RBAC?
+- Does the platform support services of type `LoadBalancer`?
+
+If one of the above questions can have multiple answers (e.g. different Kubernetes versions)
+then make the platform more specific. E.g. consider "GKE with Kubernetes 1.10.2" a platform
+instead of "GKE" which can have version "1.8", "1.9" & "1.10.2".
+
+## Platform preparations
+
+Before the tests can be run, the platform has to be prepared.
+
+### Deploy the ArangoDB operators
+
+Deploy the following ArangoDB operators:
+
+- `ArangoDeployment` operator
+- `ArangoDeploymentReplication` operator
+- `ArangoLocalStorage` operator
+
+To do so, follow the [instructions in the manual](../Manual/Deployment/Kubernetes/Usage.md).
+
+### `PersistentVolume` provider
+
+If the platform does not provide a `PersistentVolume` provider, create one by running:
+
+```bash
+kubectl apply -f examples/arango-local-storage.yaml
+```
```

Comment:
We probably want to create an arango-local-storage on all platforms, just to see if it works, and because performance will be best with local disks.